.github/workflows/compile.yml

```diff
@@ -46,6 +46,7 @@ jobs:
         with:
             registry: true
       - name: Restore Webpack cache
```

Comment:
I have discussed this with @UncleBill and I think you can remove the whole cache section.